frontend/pages/admin/IntegrationsPage/cards/Mdm/components/EditTeamModal.tsx

```diff
@@ -1,29 +1,83 @@
-import React from "react";
+import React, { useState, useContext, FormEvent } from "react";
+
+import { AppContext } from "context/app";
+import configAPI from "services/entities/config";
 
 import Modal from "components/Modal";
 import Button from "components/buttons/Button";
 
+// @ts-ignore
+import Dropdown from "components/forms/fields/Dropdown";
+
 interface IEditTeamModal {
   onCancel: () => void;
-  onEdit: () => void;
+  currentDefaultTeamName: string | undefined;
 }
 
 const baseClass = "edit-team-modal";
 
-const EditTeamModal = ({ onCancel, onEdit }: IEditTeamModal): JSX.Element => {
+const EditTeamModal = ({
+  onCancel,
+  currentDefaultTeamName,
+}: IEditTeamModal): JSX.Element => {
+  // availableTeams: Array<ITeamSummary>
+  const { availableTeams } = useContext(AppContext);
+
+  const teamNameOptions = availableTeams?.map((teamSummary) => {
+    return { value: teamSummary.name, label: teamSummary.name };
+  });
+
+  const [defaultTeamName, setDefaultTeamName] = useState<string | undefined>(
```

Comment:
the type is inferred correctly in this case and you don't need to explicitly define the generic types.

```tsx
const [defaultTeamName, setDefaultTeamName] = useState(currentDefaultTeamName)
```